File: real-time/part11-flink-job/src/main/scala/com/example/flink/bean/ProvinceStats.scala
package com.example.flink.bean

import java.math.BigDecimal

case class ProvinceStats(
                          var stt: String = null,
                          var edt: String = null,
                          var province_id: Long = 0L,
                          var province_name: String = null,
                          var area_code: String = null,
                          var iso_code: String = null,
                          var iso_3166_2: String = null,
                          var order_amount: BigDecimal = BigDecimal.ZERO,
                          var order_count: Long = 0L,
                          var ts: Long = 0L
                        )

File: real-time/part05-publish-query/src/main/java/com/example/publish/query/bean/ProvinceStats.java
package com.example.publish.query.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ProvinceStats {
    private String stt;
    private String edt;
    private String province_id;
    private String province_name;
    private BigDecimal order_amount;
    private String ts;
}